pycam/Importers/STLImporter.py

```diff
@@ -81,7 +81,7 @@ def is_binary_input_format(source):
         return True
 
 
-def ImportModel(filename, use_kdtree=True, callback=None, **kwargs):
+def ImportModel(filename, use_kdtree=True, callback=None, numfacets=None, **kwargs):
```

Comment:
This missing variable made me aware of a problem, I forgot there. I fixed it now. Thus this change is not necessary anymore.
Thank you!